lib/components/ChangeWalletNetworkNotificationBanner.jsx

```diff
@@ -7,15 +7,14 @@ import { WalletContext } from 'lib/components/WalletContextProvider'
 import { WALLETS } from 'lib/constants'
 import { NotificationBanner } from 'lib/components/NotificationBanners'
 import { NETWORK } from 'lib/utils/networks'
-import { ButtonLink } from 'lib/components/ButtonLink'
 
 export const ChangeWalletNetworkNotificationBanner = (props) => {
   const { walletConnected, walletMatchesNetwork, walletNetwork, view, chainId } = useNetwork()
 
   if (!walletConnected || walletMatchesNetwork) return null
 
   return (
-    <NotificationBanner className='bg-teal'>
+    <NotificationBanner className='bg-teal' canClose>
```

Comment:
Why do we want them to be able to close this? Currently this is the only thing that let's them know that they're on the wrong network.

Reply:
- It takes up a decent amount of space so for users will small screens / mobile
- If I know my wallet can't connect to the network it's suggesting, it's just annoying